homeassistant/components/opensprinkler/switch.py

```diff
@@ -0,0 +1,123 @@
+"""Opensprinkler integration."""
```

Comment:
When adding a new integration, please only include a single platform. I am stopping my reviewing here. Please remove all but 1 platform so we can get that merged, then have more PRs for the other ones.